src/familiar.ts

```diff
@@ -222,10 +224,34 @@ export function pocketProfessorLectures(): number {
   return 2 + Math.ceil(Math.sqrt(familiarWeight($familiar`Pocket Professor`) + weightAdjustment()));
 }
 
-function timeToMeatify(): boolean {
-  return (
-    have($familiar`Grey Goose`) &&
-    $familiar`Grey Goose`.experience >= 400 &&
-    !get("_meatifyMatterUsed")
-  );
+export function timeToMeatify(): boolean {
+  if (!have($familiar`Grey Goose`) || get("_meatifyMatterUsed") || myInebriety() > inebrietyLimit()) return false;
+  else if ($familiar`Grey Goose`.experience >= 400) return true;
+  else if (!globalOptions.ascending || myAdventures() > 50) return false;
+  
+  //Check Wanderers
+  const F = get("_sausageFights");
+  const totalTurns = totalTurnsPlayed();
+
+  const nextProtonicGhost = (have($item`protonic accelerator pack`)) ? Math.max(0, get("nextParanormalActivity") - totalTurns) : Infinity;
+  const nextVoteMonster = (have($item`"I Voted!" sticker`) && get("_voteFreeFights") < 3) ? Math.max(0, (totalTurns % 11 - 1) % 11) : Infinity;
+  const nextSausageGoblin = (have($item`Kramco Sausage-o-Matic™`)) ? Math.max(0, 4 + 3 * F + Math.pow(Math.max(0, F-5), 3) + get("_lastSausageMonsterTurn") - totalTurns) : Infinity;
+  const nextVoidMonster = (have($item`cursed magnifying glass`) && get("_voidFreeFights") < 5) ? -get("cursedMagnifyingGlassCount") % 13 : Infinity;	
```

Comment:
You'll need to somehow determine if the player will actually be wielding their CMG--you could do something like
```
get("valueOfFreeFight", 2000) / 13 > baseMeat * (usingLatte() ? 0.75 : 0.6)
```
, assuming that 0.75 is a decent estimate for latte